src/eko/matching_conditions/operator_matrix_element.py

```diff
@@ -46,19 +48,22 @@ def compute_harmonics_cache(n, order, is_singlet):
 
     """
     # max harmonics sum weight for each qcd order
-    max_weight = {1: 2, 2: 3, 3: 5}
+    max_weight = {1: 2, 2: 3, 3: 3}
+    # _N3LO_ max_weight = {1: 2, 2: 3, 3: 5}
     # max number of harmonics sum of a given weight for each qcd order
-    n_max_sums_weight = {1: 1, 2: 3, 3: 7}
+    n_max_sums_weight = {1: 1, 2: 3, 3: 3}
```

Comment:
Actually, why you changed the numbers?

Reply:
Ok, but I believe here `3` is actually N3LO, so you would never use in any case.

Reply:
True, but this would require further adjusting of the tests